controller_manager/scripts/controller_group

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# Provides quick access to the services exposed by controller_manager_msgs
+import rospy
+from argparse import ArgumentParser
+from controller_manager import controller_manager_interface
+from controller_manager_msgs.srv import ListControllers, ListControllersRequest
+
+
+class Groups(object):
```

Comment:
Since this class has no state, IMO it's superfluous. Consider moving the methods up to module level (eg `list_groups`, `spawn_group`, etc).

Reply:
sure